code/game/objects/structures/watercloset.dm

```diff
@@ -44,6 +45,17 @@
 	icon_state = "toilet[open][cistern]"
 
 /obj/structure/toilet/attackby(obj/item/I as obj, mob/living/user as mob)
+	if(open && cistern && !state && istype(I,/obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(R.amount < 2) return
+		user << "<span class='notice'>You add the rods to the toilet.</span>"
+		R.use(2)
+		state++
```

Comment:
I don't really like how states are used like this, could you change these to be defines so we can know exactly what state an object is in?